low-level/files/etc/logrotate.d/sspl_sb_logs

```diff
@@ -0,0 +1,13 @@
+/var/log/cortx/sspl/sspl_support_bundle.log
+{
+    size 2M
+    rotate 5
+    compress
+    dateext
+    dateformat -%Y-%m-%d-%s.log
+    copytruncate
+    missingok
+    postrotate
+        /bin/kill -HUP `cat /var/run/syslogd.pid 2> /dev/null` 2> /dev/null || true
+    endscript
+}
```

Comment:
Which attribute does define the maximum number of compressed/archived files to be retained?
Otherwise keeping very old files or outdated files will consume more space in the drive.

Reply:
#L4 rotate 5
keeping max 5 support bundle log files as discussed with chetan.
Any change required?

Reply:
Fine. Got it.

Reply:
Got it.